src/patient-registration/input/basic-input/select/select-input.component.tsx

```diff
@@ -1,50 +1,26 @@
 import React from 'react';
 import { useField } from 'formik';
 import styles from './../../input.scss';
+import { Select, SelectItem } from 'carbon-components-react';
 
 interface SelectInputProps {
   name: string;
   options: Array<string>;
   label: string;
-  showRequiredAsterisk?: boolean;
 }
 
-export const SelectInput: React.FC<SelectInputProps> = ({ name, options, label, showRequiredAsterisk }) => {
+export const SelectInput: React.FC<SelectInputProps> = ({ name, options, label }) => {
   const [field, meta] = useField(name);
   const selectOptions = [
-    <option key="" value="" disabled>
-      Select {label}
-    </option>,
-    options.map(currentOption => (
-      <option key={currentOption} value={currentOption}>
-        {currentOption}
-      </option>
-    )),
+    <SelectItem disabled hidden text={`Select ${label}`} key="" value="" />,
+    ...options.map(currentOption => <SelectItem text={currentOption} value={currentOption} key="" />),
```

Comment:
Is it wanted to have all with the key `""`?

Reply:
Hmm, will look into why it's that way, I didn't modify that particular piece of code. I just switched to carbon components while maintaining the existing implementation.